gridt/src/app/movements/movements.page.html

```diff
@@ -1,73 +1,77 @@
-      <ion-col size="12" size-sm="8" offset-sm="2" text-center>
-        <p>There are no movements  right now, please come back later!</p>
-      </ion-col>
-    </ion-row>
-  </ion-grid>
+  <!-- Then print a list of all the movements -->
+  <ion-list>
+    <ion-grid *ngIf="!(movements$ | async)">
+      <ion-row>
+        <ion-col size="12" size-sm="8" offset-sm="2" text-center>
+          <p>There are no movements right now, please come back later!</p>
+        </ion-col>
+      </ion-row>
+    </ion-grid>
     <ion-grid *ngIf="movements$ | async">
-
-  <!-- Create a list item for each search result.
+      <!-- Create a list item for each search result.
     Here I had to guess the routerLink, but that can be changed later.
     async allows the results to change by making a new search.
   -->
       <ion-row>
         <ion-col size-md="6" offset-md="3">
-          <ion-item *ngFor="let movement of movements$ | async | movementFilter: searchText"
-            [routerLink]="['/', 'movements', movement.name]" 
-            detail button
+          <ion-item
+            *ngFor="let movement of movements$ | async | movementFilter: searchText"
+            [routerLink]="['/', 'movements', movement.name]"
+            detail
+            button
           >
             <!-- Right now, this just shows every single movement -->
             <!-- Display the title on the left with icon if user is subscribed-->
             <ion-label text-wrap>
               <h3>
-                {{ movement.name }} 
-                <ion-icon *ngIf="movement.subscribed" name="people-sharp"></ion-icon> 
+                {{ movement.name }}
+                <ion-icon
+                  *ngIf="movement.subscribed"
+                  name="people-sharp"
+                ></ion-icon>
               </h3>
-              <p class="shortdescription">{{ movement.short_description }}</p> 
+              <p class="shortdescription">{{ movement.short_description }}</p>
             </ion-label>
 
-             <!-- Display the interval (like, weekly, monthly etc.) on the right -->
-            <ion-label slot="end"> 
-              {{ movement.interval }}
-            </ion-label>
+            <!-- Display the interval (like, weekly, monthly etc.) on the right -->
+            <ion-label slot="end"> {{ movement.interval }} </ion-label>
           </ion-item>
-         </ion-col>
+        </ion-col>
       </ion-row>
     </ion-grid>
+  </ion-list>
+  <!-- Custom floating button for adding movements -->
+  <ion-fab vertical="bottom" horizontal="end" slot="fixed">
+    <ion-fab-button class="custom" color="primary" routerLink="/add">
+      Add Movement <ion-icon name="add"></ion-icon>
+    </ion-fab-button>
+  </ion-fab>
 
-</ion-list>
-
+  <!-- Regular floating button
+    <ion-fab vertical="bottom" horizontal="end" slot="fixed">
+      <ion-fab-button color="primary" routerLink="/add">
+        <ion-icon name="add"></ion-icon>
+      </ion-fab-button>
+    </ion-fab> -->
```

Comment:
I personally prefer this one. If we wanted to go for absolute clarity we could implement something like: 
press once => show add movement text => press twice go to add movements page
but that smells like over engineering to me.